deployments/ecs/README.md

```diff
@@ -0,0 +1,60 @@
+# ECS Deployment
+
+## Create Task Definition
+To deploy the agent on AWS Elastic Container Service (ECS) you must first
+create the agent task definition.  To do this using the web admin console:
+
+ 1. Go to your ECS web admin console and go to the "Task Definitions" tab.
+ 2. Click on "Create new Task Definition".
+ 3. Select the big "EC2" square and click "Next step".  The agent only supports
+	EC2 mode and not Fargate at this time.
+ 4. Scroll to the bottom of the page and click on "Configure via JSON".
+ 5. Paste in the contents of the file [signalfx-agent-task.json](./signalfx-agent-task.json)
+	and click "Save".
+ 6. Click on the "signalfx-agent" container definition under "Container
+	Definitions" and find the section on environment variables.
+ 7. Change the value of the envvar `ACCESS_TOKEN` to the access token of the
+	SignalFx organization to which you wish to send metrics.
+ 8. Click "Update" and finally "Create" at the bottom of the task definition
+	input form to create the task definition.
+
+You can also do this with the AWS CLI tool by issuing the following command:
+
+`aws ecs register-task-definition --cli-input-json file:///path/to/signalfx-agent-task.json`
+
+## Launching the Agent
+The agent is designed to be run as a Daemon service in an EC2 ECS cluster.
+
+To create an agent service from the ECS web admin console:
+
+ 1. Go to your cluster in the web admin
+ 2. Click on the "Services" tab.
+ 3. Click "Create" at the top of the tab.
+ 4. Select:
+     - `Launch Type` -> `EC2`
+	 - `Task Definition (Family)` -> `signalfx-agent`
+	 - `Task Definition (Revision)` -> `1` (or whatever the latest is in your case)
+	 - `Service Name` -> `signalfx-agent`
+	 - `Service type` -> `DAEMON`
+ 5. Leave everything else at default and click "Next step"
+ 6. Leave everything on this next page at their defaults and click "Next step".
+ 7. Leave everything on this next page at their defaults and click "Next step".
```

Comment:
duplicate doc line

Reply:
There are actually two screens that have the same instructions.